src/control/cmd/daos/container.go

```diff
@@ -397,76 +394,50 @@ func (cmd *containerCreateCmd) contCreate() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	contID := cmd.contUUID.String()
+	cContID := C.CString(contID)
+	defer freeString(cContID)
 
-	var contID string
-	if cmd.contUUID == uuid.Nil {
-		contID = cmd.contLabel
-	} else {
-		contID = cmd.contUUID.String()
-	}
-
-	cmd.Infof("Successfully created container %s", contID)
-	return contID, nil
-}
-
-func (cmd *containerCreateCmd) contCreateUNS() (string, error) {
-	var dattr C.struct_duns_attr_t
-
-	props, cleanupProps, err := cmd.getCreateProps()
-	if err != nil {
-		return "", err
+	cleanupContainer := func() {
+		rc := C.daos_cont_destroy(cmd.cPoolHandle, cContID, goBool2int(true), nil)
+		if err := daosError(rc); err != nil {
+			cmd.Noticef("Failed to clean-up container %v", err)
```

Comment:
Should probably be an error, right?

Reply:
Indeed, I hesitated between error and notice.  Finally I decided to use notice as we already going to print an error message when this function is called.  Thus, using notice was to get focus on the real error triggering this cleanup function and not one of the side effect of this real error.
However, if you think that an error message is more suited, I  have no issue to update it.

Reply:
The rule of thumb I use is to ask what happens if the user misses the message. In this case, I guess they might be confused if they try to recreate the container and get an error, but it's not the end of the world. TBH, these kinds of decisions were why I initially only added DEBUG, INFO and ERROR levels to the logger. :)

I just double-checked, and NOTICE messages do go to stderr, so my concern that the message might be suppressed in JSON-output mode has been allayed.